spec/models/labware_creators/common_file_handling/csv_file_for_tube_rack_spec.rb

```diff
@@ -178,19 +196,22 @@
 
   context 'A file which has missing values' do
     let(:file) do
-      fixture_file_upload('spec/fixtures/files/tube_rack_scan_with_missing_values.csv', 'sequencescape/qc_file')
+      fixture_file_upload(
+        'spec/fixtures/files/common_file_handling/tube_rack/tube_rack_scan_with_missing_values.csv',
+        'sequencescape/qc_file'
+      )
     end
 
     describe '#valid?' do
       it 'should be invalid' do
         expect(subject.valid?).to be false
       end
 
-      let(:row4_error) { 'Tube rack scan barcode cannot be empty, in row 4 [C1]' }
+      let(:row3_error) { 'Tube rack scan tube barcode cannot be empty, in row 3 [C1]' }
```

Comment:
I am trying to find out what caused the change in row number here. The lines in the fixture file did not change recently.

Reply:
It seems there was a header in the file before:
Position,Barcode